docs/installing_wash.md

```diff
@@ -11,12 +11,24 @@ Wash is distributed as a single binary; the only prerequisite is `libfuse`. Here
 
 * Install the Wash binary
     * On MacOS using homebrew: `brew install puppetlabs/puppet/wash`
-    * or [download](https://github.com/puppetlabs/wash/releases/latest) Wash for your platform, then run the following commands in your terminal:
+    * or download Wash for your platform.
+        * On Linux:
+            ```
+            curl -sfLO https://github.com/puppetlabs/wash/releases/download/{WASH_VERSION}/wash-{WASH_VERSION}-x86_64-unknown-linux.tgz
+            ```
+        * On MacOS:
+            ```
+            curl -sfLO https://github.com/puppetlabs/wash/releases/download/{WASH_VERSION}/wash-{WASH_VERSION}-x86_64-apple-darwin.tgz
+            ```
+
+      where `{WASH_VERSION}` is the [latest Wash version](https://github.com/puppetlabs/wash/releases/latest) (e.g. `0.16.0`). After downloading the `.tgz`, run the following commands:
         * `tar -xvzf <path_to_downloaded_wash_tgz>` (unpack it)
-        * `chmod +x <path_to_wash>` (ensure it's executable)
```

Comment:
I noticed that `tar -xvzf` places the extracted `wash` binary in the dir. that you ran the command, which is why I made this change.